libs/hdf-converters/test/utils.ts

```diff
@@ -64,3 +64,11 @@ export function omitHDFTimes(
     })
   };
 }
+
+export function omitParentProfileVersion(
```

Comment:
unnecessary - combine into the other function

Reply:
Done